logging/google/cloud/logging/logger.py

```diff
@@ -379,13 +410,17 @@ def log_text(self, text, labels=None, insert_id=None, severity=None,
 
         :type timestamp: :class:`datetime.datetime`
         :param timestamp: (optional) timestamp of event being logged.
+
+        :type resource :class:``google.cloud.logging.resource`
+        :param resource: (Optional) Monitored resource of the entry
         """
         self.entries.append(
             ('text', text, labels, insert_id, severity, http_request,
-             timestamp))
+             timestamp, resource))
```

Comment:
ISTM that this is incorrect.

You should be allowing `self.resource` to matter if the caller never sends a `resource`